test/e2e/specs/fakerp/prometheus.go

```diff
@@ -57,10 +59,15 @@ var _ = Describe("Prometheus E2E tests [Fake]", func() {
 		req.Header.Add("Authorization", "Bearer "+string(token))
 
 		cli := &http.Client{
-			Transport: &http.Transport{
-				TLSClientConfig: &tls.Config{
-					InsecureSkipVerify: true,
+			Transport: &kubeclient.RetryingRoundTripper{
```

Comment:
why is this being changed? did you mean to change it to AuthorizingRoundTripper?

Reply:
my bad attempt at fixing the issue of the test not working right after provisioning a new cluster. Rolled back.